### PR TITLE
Update Fields - Calculated Fields - Lookup & Reference of user document

### PR DESCRIPTION
**Pull request by recca5p**

### CODE REVIEW

File: docs/user/fields/calculated/lookup-reference.md [hunk 1]
```diff
@@ -34,7 +36,7 @@ for you to configure:
 
 ### Sample
 
-The following is an extract of the HR Dataset 2016.xlsx "Employees"
+The following is an extract of the [HR Dataset 2016](http://download.infragistics.com/reportplus/help/samples/HR%20Dataset_2016.xlsx) "Employees"
```
Comment (hainv-ohio) [inline docs/user/fields/calculated/lookup-reference.md:39]: Use local file

File: docs/user/fields/calculated/lookup-reference.md [hunk 2]
```diff
@@ -10,8 +10,10 @@ Lookup and reference fields will allow you to work with your current
 spreadsheet and dashboard, returning text references to cells, rows and
 dashboard variables.
 
->[!NOTE] All samples included in the table below were created with the **HR Dataset 2016.xlsx** spreadsheet.
-
+:::note
+All samples included in the table below were created with the **[HR Dataset 2016](http://download.infragistics.com/reportplus/help/samples/HR%20Dataset_2016.xlsx)
```
Comment (hainv-ohio) [inline docs/user/fields/calculated/lookup-reference.md:14]: Use local file

**Review by hainv-ohio — CHANGES_REQUESTED** [review]

**Review by hainv-ohio — CHANGES_REQUESTED** [review]
image
Add breaks

File: docs/user/fields/calculated/lookup-reference.md (hunk 2 above)
Comment (hainv-ohio) [inline docs/user/fields/calculated/lookup-reference.md:14]: Formating is not being applied: 
image

**Review by hainv-ohio — CHANGES_REQUESTED** [review]

File: docs/user/fields/calculated/lookup-reference.md (hunk 2 above)
Comment (recca5p) [reply docs/user/fields/calculated/lookup-reference.md]: Done

Comment (recca5p) [reply docs/user/fields/calculated/lookup-reference.md]: Done

File: docs/user/fields/calculated/lookup-reference.md (hunk 1 above)
Comment (recca5p) [reply docs/user/fields/calculated/lookup-reference.md]: Done

**Review by hainv-ohio — APPROVED** [review]

**Pull request merged**